mk/spksrc.native-env.mk

```diff
@@ -3,8 +3,6 @@ PKG_CONFIG_LIBDIR = $(INSTALL_DIR)/$(INSTALL_PREFIX)/lib/pkgconfig
 INSTALL_DIR = $(WORK_DIR)/install
 INSTALL_PREFIX = /usr/local
 
-ENV += -u LDSHARED
-ENV += -u MAKEFLAGS
-ENV += -u PKG_CONFIG -u PKG_CONFIG_LIBDIR -u PKG_CONFIG_PATH
+ENV := -u LDSHARED -u PKG_CONFIG -u PKG_CONFIG_LIBDIR -u PKG_CONFIG_PATH $(ENV)
```

Comment:
I think `+=` is easier to understand than `:=`. Any reason for this change?

Reply:
Reason is, all the `-u` must be always at first as otherwise it pseudo-silently fails.  As such `:=` becomes the only way to ensure that.

I'll add a comment in the code for this...

Reply:
Thanks
IMHO `unset variables` sounds better, but it doesn't matter that much

Reply:
done with https://github.com/SynoCommunity/spksrc/pull/4857

Reply:
I'm not convinced, at least this wrong for Erlang. Both `ENV +=` and `ENV :=` work the same for this package. Only -u MAKEFLAGS was missing

Reply:
Fix is in #4858

Reply:
What needs to be happening for the later call is this:
```
env -u LDSHARED -u PKG_CONFIG -u PKG_CONFIG_LIBDIR -u PKG_CONFIG_PATH ...
```
But, as `PARALLEL_MAKE` was previously added to the environment prior to that, it becomes the first item in the list such as:
```
env PARALLEL_MAKE=max -u LDSHARED -u PKG_CONFIG -u PKG_CONFIG_LIBDIR -u PKG_CONFIG_PATH ...
```
And this ends-up throwing an error about the `-u` as unsetting variable needs to be first to the `env` call...

Question could then be, does `PARALLEL_MAKE` really needs to be part of `$(ENV)` or not?  If not then we could switch back to `+=` instead of `:=`.  Although I don't recall the exact reason why it needed to be into `$(ENV)`... perhaps something worth investigating?

Reply:
Ah ok thanks for the explanation :+1: we can leave it with `:=` then.